internal/consensus/reactor.go

```diff
@@ -885,29 +903,13 @@ func pickPartForCatchup(
 	return part
 }
 
-func pickVoteToSend(
-	logger log.Logger,
+func sendEntireCommit(logger log.Logger,
 	conS *State,
 	rs *cstypes.RoundState,
 	ps *PeerState,
 	prs *cstypes.PeerRoundState,
 	rng *rand.Rand,
-) *types.Vote {
-	// If height matches, then send LastCommit, Prevotes, Precommits.
-	if rs.Height == prs.Height {
-		heightLogger := logger.With("height", prs.Height)
-		return pickVoteCurrentHeight(heightLogger, rs, prs, ps, rng)
-	}
-
-	// Special catchup logic.
-	// If peer is lagging by height 1, send LastCommit.
-	if prs.Height != 0 && rs.Height == prs.Height+1 {
-		if vote := ps.PickVoteToSend(rs.LastCommit, rng); vote != nil {
-			logger.Debug("Picked rs.LastCommit to send", "height", prs.Height)
-			return vote
-		}
-	}
-
+) *types.VoteSetReader {
```

Comment:
```suggestion
) types.VoteSetReader {
```
As `VoteSetReader` is an interface, there's no point in defining a pointer